packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -85,15 +89,37 @@ export class ModularChangeFamily
                 return convertGenericChange(
                     genericChange,
                     handler,
-                    this.childComposer,
+                    (children) => this.composeNodeChanges(children, genId),
+                    genId,
                 ) as FieldChangeset;
             }
             return change.change;
         });
         return { fieldKind, changesets: normalizedChanges };
     }
 
-    compose(changes: TaggedChange<FieldChangeMap>[]): FieldChangeMap {
+    compose(changes: TaggedChange<ModularChangeset>[]): ModularChangeset {
+        let maxId = changes.reduce((max, change) => Math.max(change.change.maxId, max), -1);
+        const genId: IdAllocator = () => {
+            const id = maxId;
+            maxId += 1;
+            return brand(id);
+        };
```

Comment:
Doesn't this allocate `-1` as the first ID?

Reply:
Yes. I'm adding some tests now and I caught this bug.

Reply:
No good reason. I think mostly because we were using 0 as the first OpId. I will at least change it so that we use `undefined` instead of `-1`.